### DISCUSSION TITLE
Action plan empty

### DISCUSSION THREAD

**Discussion by anthoshell91** [dbarzin/deming#85] [Q&A]
Hello guys
Can someone help me ? Action plan is still empty... but i can find them in the controls. 
![image](https://github.com/dbarzin/deming/assets/86150595/fe7ccf27-bae1-48cd-8126-7991c1ca2739)
![image](https://github.com/dbarzin/deming/assets/86150595/479c25ee-29a2-4875-8ab9-73f7ef9310ac)
Thanks a lot.

**Comment by dbarzin** [comment]
An action plan is present if a measure has been made, and the score is red or orange.
